source/run-command.txt

```diff
@@ -0,0 +1,182 @@
+.. _c-run-command:
+
+======================
+Run a Database Command
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :description: Learn how to use the C Driver to run a database command.
+   :keywords: command, server, usage, code example
+
+Overview
+--------
+
+In this guide, you can learn how to run a database command with the
+{+driver-short+}. You can use database commands to perform a variety of
+administrative and diagnostic tasks, such as fetching server statistics,
+initializing a replica set, or running an aggregation pipeline.
+
+.. important:: Prefer Driver Methods to Database Commands
+
+   The driver provides wrapper methods for many database commands. We recommend
+   using driver methods instead of executing database commands when possible.
+   
+   To perform administrative tasks, use the :mongosh:`MongoDB Shell </>` instead
+   of the {+driver-short+}. Calling the ``db.runCommand()`` method inside the
+   shell is the preferred method to issue database commands, as it provides a
+   consistent interface between the shell and drivers.
+
+Execute a Command
+-----------------
+
+To run a database command, you must specify the command and any relevant
+parameters in a BSON document, then pass this BSON document to a command
+execution method. The {+driver-short+} provides the following methods to run
+database commands:
+
+- ``mongoc_client_command_simple()``, which returns the first document from the
+  result cursor into ``reply``. This method is a simplified interface that sends
```

Comment:
I'm a little confused at what this means still. What is it returning documents from? and what is a ``reply``? Can this maybe be rephrased or expanded on a bit?

Reply:
Rewording!